src/netius/test/base/conn.py

```
@@ -121,6 +121,196 @@ def test_close_reason_flush(self):
         self.assertEqual(connection.status, conn.CLOSED)
         self.assertEqual(connection.close_reason, netius.REASON_TIMEOUT)
 
+    def test_upgrade(self):
+        if mock == None:
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)